tests/runtime/test_cluster_creation.py

```diff
@@ -12,5 +12,9 @@ def test_default_cluster_spinup_time(auto_benchmark_time):
     with Cluster(
         name=f"test_default_cluster_spinup_time-{uuid.uuid4().hex[:8]}",
         package_sync=True,
-    ):
+    ) as cluster:
+        test_run_benchmark.cluster_name = cluster.name
+        test_run_benchmark.cluster_id = cluster.cluster_id
+        test_run_benchmark.cluster_details_url = cluster.details_url
+
         pass
```

Comment:
Now that there is content here, we don't need the `pass`. But I'm not worried about it.